cmds/core/wc/wc.go

```diff
@@ -92,8 +92,18 @@ func command(stdin io.Reader, stdout io.Writer, stderr io.Writer, p params, args
 	}
 }
 
+// TODO: if we change the behavior to return an error, then inputting multiple
+// non-existing files will result in a premature exit and the behavior will
+// differ from the original implementation. This should be adjusted. Until then
+// disable the noparam linter to fail because we always return nil.
+//
+//nolint:unparam
 func (c *cmd) run() error {
-	var totals cnt
+	var (
+		totals cnt
+		err    error
```

Comment:
I really fail to see the point of pre-declaring err this way. You're putting declarations far from use, and I can tell you, that's caused trouble in production code. Is this a golangci-lint thing? If so I don't agree.